### PR TITLE
Fix possible typo

### PR DESCRIPTION
**Pull request by sapegin**
I have nothing against discussing burritos at night but I believe it’s a typo. Feel free to close if I’m wrong ;-)

### CODE REVIEW

**Comment by bkeepers** [comment]
Haha. I'm think this is intended to be "discussing", as in, discussing how you like your burritos (carne asada and black beans) and where is the best place to get them. Does that make sense?

Maybe it'd be more clear to change it to:

> …or talking about burritos late at night over chat

**Comment by sapegin** [comment]
That “over chat” is what confused my non-native ear, so I’ve read it like “talking about burritos while talking” ;-)

So maybe something like?

> …or talking about burritos late at night over a beer

**Comment by bkeepers** [comment]
> That “over chat” is what confused my non-native ear, so I’ve read it like “talking about burritos while talking” ;-)

Ahh, I think "over chat" is about online chat (IRC, clack, etc.) Is there a better way to make that clear?

**Comment by sapegin** [comment]
Ah, than maybe replace “chat” with “Slack” or “IRC”?

**Comment by bkeepers** [comment]
went with:

> … late night online chats about burritos

File: _articles/how-to-contribute.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Whether it's coding, user interface design, graphic design, writing, or organizi
 
 ### Meet people who are interested in similar things
 
-Open source projects with warm, welcoming communities keep people coming back for years. Many people form lifelong friendships through their participation in open source, whether it's running into each other at conferences or eating burritos late night over chat.
+Open source projects with warm, welcoming communities keep people coming back for years. Many people form lifelong friendships through their participation in open source, whether it's running into each other at conferences or late night online chats about burritos.
```
Comment (sapegin) [inline _articles/how-to-contribute.md]: 🌯

**Comment by sapegin** [comment]
That’s clear enough, thanks!

**Pull request merged**